### DISCUSSION TITLE
How to be white list only?

### DISCUSSION THREAD

**Discussion by playgithub** [DNSCrypt/dnscrypt-proxy#1744] [Q&A]
e.g.

Only `github.com` should be through proxy, other domain names works as if no `dnscrypt-proxy`,
and how can I check if this is true?

Thanks

**Comment by jedisct1** [comment]
https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Filters#allowlist-only

**Reply by playgithub** [reply]
It take effects, but I have some questions and problems:
* I have not set the alternative dns server, how does the names not in allowed list be resolved?
* `dnscrypt-proxy` as a service, `github.com` still can not be accessed sometimes, what's the problem? And `nslookup github.com` & `dnscrypt-proxy -resolve github.com` return different ip addresses, it seems not as expected.

A Case
```
λ dnscrypt-proxy.exe -resolve github.com
...
IPv4 addresses: 13.250.177.223
...
λ nslookup github.com
...
Address:  13.229.188.59
...
```
OS: Windows, only one dns server is set, which is `127.0.0.1`,<br>
In chrome "Developer Tools (F12)", the ip address is 13.229.188.59, why doesn't `dnscrypt-proxy` take effect?


Thanks
